### DISCUSSION TITLE
Homebrew not installing on Oracle Linux 7.9

### DISCUSSION THREAD

**Discussion by blaqoso** [Homebrew/discussions#3179] [Getting started]
I am trying to install homebrew on Oracle Linux 7.9. 

My git version is 2.70. My curl version is 7.29. This is the error:

/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Found Git: /usr/local/git/bin/git
The version of cURL that was found does not satisfy requirements for Homebrew.
Please install cURL 7.41.0 or newer and add it to your PATH.

Is there a way to get an older version of homebrew that will install with curl 7.29?

**Comment by SMillerDev** [comment]
See for example: https://github.com/Homebrew/discussions/discussions/2215

**Comment by blaqoso** [comment]
Thanks. I did what was suggested:

 echo $HOMEBREW_DEVELOPER; echo $HOMEBREW_GIT_PATH; echo $HOMEBREW_CURL_PATH
1
/usr/local/git/bin/git
/bin/curl

Response: 
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Found Git: /usr/local/git/bin/git
The version of cURL that was found does not satisfy requirements for Homebrew.
Please install cURL 7.41.0 or newer and add it to your PATH.

**Comment by blaqoso** [comment]
I decided to update curl to curl-7.41:

wget https://curl.haxx.se/download/curl-7.67.0.tar.gz
gunzip -c curl-7.67.0.tar.gz | tar xvf -
cd curl-7.41.0
./configure --with-ssl

This is where it hangs:

checking if gethostbyname_r is prototyped... yes
checking if gethostbyname_r takes 3 args.... no
checking if gethostbyname_r takes 5 args.... no
checking if gethostbyname_r takes 6 args.... yes
checking if gethostbyname_r is compilable... yes
checking if gethostbyname_r usage allowed... yes
checking if gethostbyname_r might be used... yes
checking if gethostname can be linked... yes
checking if gethostname is prototyped... yes
checking if gethostname is compilable... yes
checking for gethostname arg 2 data type... size_t
checking if gethostname usage allowed... yes
checking if gethostname might be used... yes
checking if getifaddrs can be linked... yes
checking if getifaddrs is prototyped... yes
checking if getifaddrs is compilable... yes
**checking if getifaddrs seems to work...**
